Source/JavaScriptCore/builtins/AsyncFromSyncIteratorPrototype.js

```diff
@@ -180,4 +199,9 @@ function AsyncFromSyncIterator(syncIterator, nextMethod)
 
     @putByIdDirectPrivate(this, "syncIterator", syncIterator);
     @putByIdDirectPrivate(this, "nextMethod", nextMethod);
+    @putByIdDirectPrivate(this, "onRejected", function (error, promise) {
+        "use strict";
+
+        @asyncFromSyncIteratorOnRejected(syncIterator, error, promise);
+    });
```

Comment:
@Constellation  I've fixed to create a closure only once inside AsyncFromSyncIterator constructor. What do you think?